app/views/registration/steps/_finished.html.haml

```diff
@@ -9,8 +9,4 @@
             %p {{'registration_finished_login' | t}}
     .row
       .small-12.columns.text-center
-        %h4{ "ng-bind" => "'registration_finished_activate' | t:{enterprise: enterprise.name}" }
```

Comment:
We can remove these translation keys right? `registration_finished_activate` and `registration_finished_activate_instruction_html ` no longer used?

Reply:
Done.